src/core/Akka.Persistence/persistence.conf

```diff
@@ -77,21 +77,21 @@ akka.persistence {
         # Dispatcher used by every plugin which does not declare explicit
         # `plugin-dispatcher` field.
         default-plugin-dispatcher {
-            type = PinnedDispatcher
-            executor = "fork-join-executor"
+            type = Dispatcher
+            executor = "default-executor"
```

Comment:
Moves Akka.Persistence back over to the normal dispatcher when there isn't a custom one used by the plugin (there are none, IIRC)